test/runtests.jl

```diff
@@ -1,11 +1,13 @@
-using FixedPointNumbers, Test
+using FixedPointNumbers, Test, Aqua
 
 if VERSION >= v"1.6.0-DEV.816" # JuliaLang/julia #36962 # FIXME
     @test isempty(detect_ambiguities(FixedPointNumbers))
 else
     @test isempty(detect_ambiguities(FixedPointNumbers, Base, Core))
 end
 
+Aqua.test_all(FixedPointNumbers)
```

Comment:
IIRC, Aqua includes the test for ambiguity, doesn't it?

Reply:
Ohh, that's right. Updated in 40c5e55. (Removing `isempty(detect_ambiguities(FixedPointNumbers))` was my first motivation for this PR :joy:)